src/components/navigation/navigation.tsx

```diff
@@ -23,17 +22,15 @@ type Level = "primary" | "secondary" | "tertiary";
 })
 
 /**
- * @slot logo - A slot for adding a `calcite-logo` component to the primary nav level.
- * @slot user - A slot for adding a `calcite-user` component to the primary nav level.
- * @slot progress - A slot for adding a `calcite-progress` component to the primary nav level.
- * @slot nav-action - A slot for adding a `calcite-action` component to the primary nav level.
- * @slot primary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the primary nav level.
```

Comment:
Changing slot names seems like a breaking change to me. Can we continue to support prior slots here until the next major version?

Reply:
This is just a PR into the other open PR, so hasn’t made it into the codebase yet :)